src/_data/blogs.js

```diff
@@ -92,39 +91,80 @@ module.exports = async () => {
     });
 
 
+    // Get new authors
+    const newAuthors = await getData(`https://api.github.com/repos${CMS_REPO}/contents/content/authors`);
```

Comment:
Not a biggy but might want to move the slash into here, rather than the var, makes this var hard to read

Reply:
Yeah, that's a good shout - thanks @252afh